From 4649074c81beaf4ac117d913de8e623db09038b9 Mon Sep 17 00:00:00 2001
From: KO Myung-Hun <komh@chollian.net>
Date: Fri, 26 Jan 2018 14:55:58 +0900
Subject: [PATCH 14/28] Revert "qt: Fix crash when switching to fullscreen with
 video undocked"

This reverts commit d79907a5e1da092cb9ae271cba7673262ff878bf.
---
 modules/gui/qt/components/controller.cpp | 10 +++-------
 1 file changed, 3 insertions(+), 7 deletions(-)

diff --git a/modules/gui/qt/components/controller.cpp b/modules/gui/qt/components/controller.cpp
index 132b42fe16..8a9acc3bb6 100644
--- a/modules/gui/qt/components/controller.cpp
+++ b/modules/gui/qt/components/controller.cpp
@@ -877,16 +877,12 @@ void FullscreenControllerWidget::restoreFSC()
             return;
 
         QRect currentRes = QApplication::desktop()->screenGeometry( targetScreen() );
-        QWindow *wh = windowHandle();
-        if ( wh != Q_NULLPTR )
-        {
 #ifdef QT5_HAS_WAYLAND
-            if ( !b_hasWayland )
-                wh->setScreen(QGuiApplication::screens()[targetScreen()]);
+        if ( !b_hasWayland )
+            windowHandle()->setScreen(QGuiApplication::screens()[targetScreen()]);
 #else
-            wh->setScreen(QGuiApplication::screens()[targetScreen()]);
+        windowHandle()->setScreen(QGuiApplication::screens()[targetScreen()]);
 #endif
-        }
 
         if( currentRes == screenRes &&
             QApplication::desktop()->screen()->geometry().contains( previousPosition, true ) )
-- 
2.22.0

